libs/elastic-explorer/src/lib/modules/search/components/filters-form/filters-form.component.scss

```
@@ -0,0 +1,10 @@
+.my-fullScreen {
```

```suggestion
.my-fullscreen {
```
or even .my-full-screen, but we should not mix kebab-case and camelCase :)

---

and the very next class is defined in kebab-case 🙄